lib/libs/email/content/withdrawConfirmation/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Authority, CommonEmailVariables, EmailAddresses } from "shared-types";
+import { AuthoritiesWithUserTypesTemplate } from "../..";
+import { WaiverStateEmail } from "./emailTemplates";
+import { render } from "@react-email/render";
+
+export const withdrawConfirmation: AuthoritiesWithUserTypesTemplate = {
+  [Authority["1915b"]]: {
+    state: async (variables: any & CommonEmailVariables & { emails: EmailAddresses }) => {
```

Comment:
```suggestion
    state: async (variables: Events['WithdrawRequest'] & CommonEmailVariables & { emails: EmailAddresses }) => {
```

Should we use the `WithdrawRequest` obj instead of `any`?

Reply:
Same goes for the bottom on line `17`